simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -40,6 +40,27 @@
     -x -22.3 -y 14.16 -z -4.47 -Z -1.57 -model lid"
     output="screen" />
 
+  <!-- Attempting to spawn the start gate -->
+  <node
+      name ="start_gate" pkg="gazebo_ros" type="spawn_model"
+      args="-file $(find sub8_gazebo)/models/start_gate/start_gate.sdf -sdf
```

Comment:
I think in the near future, we may want to consider removing the old `sub8_simulation` package and renaming `sub8_gazebo` to `sub8_simulation`. Just an idea for now. Thoughts, @DSsoto?